### PR TITLE
Add Clients tests

### PR DESCRIPTION
**Pull request by Aboullos**
## Brief Description
Add Clients tests

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]

File: cypress/integration/clients_test.spec.ts [hunk 1]
```diff
@@ -295,6 +321,19 @@ describe("Clients test", () => {
...
+      const initialAccessTokenTab = new InitialAccessTokenTab();
+      initialAccessTokenTab
+        .goToInitialAccessTokenTab()
+        .shouldBeEmpty()
+        .goToCreateFromEmptyList()
+        .fillNewTokenData(0, 0)
+        .checkExpirationGreaterThanZeroError()
+        .checkCountValue(1)
+        .checkSaveButtonIsDisabled();
+      cy.debug();
```
Comment (mfrances17) [inline cypress/integration/clients_test.spec.ts:334]: should remove debugger

**Review by mfrances17 — CHANGES_REQUESTED** [review]
LGTM, i would remove debugger then approved

File: cypress/integration/clients_test.spec.ts (hunk 1 above)
Comment (Aboullos) [reply cypress/integration/clients_test.spec.ts:334]: Oh, right!

Comment (Aboullos) [reply cypress/integration/clients_test.spec.ts]: Resolved

**Pull request merged**